src/dynarec/la64/dynarec_la64_660f.c

```diff
@@ -628,13 +628,27 @@ uintptr_t dynarec64_660F(dynarec_la64_t* dyn, uintptr_t addr, uintptr_t ip, int
                     GETGX_empty(q0);
                     VSLLWIL_D_W(q0, q1, 0);
                     break;
+                case 0x28:
+                    INST_NAME("PMULDQ Gx, Ex");
+                    nextop = F8;
+                    GETGX(q0, 1);
+                    GETEX(q1, 0, 0);
+                    VMULWEV_D_W(q0, q0, q1);
+                    break;
                 case 0x29:
                     INST_NAME("PCMPEQQ Gx, Ex"); // SSE4 opcode!
                     nextop = F8;
                     GETEX(q1, 0, 0);
                     GETGX_empty(q0);
                     VSEQ_D(q0, q0, q1);
                     break;
+                case 0x2A:
+                    INST_NAME("MOVNTDQA Gx, Ex");
+                    nextop = F8;
+                    GETGX(q0, 1);
+                    GETEX(q1, 0, 0);
+                    VOR_V(q0, q1, q1);
+                    break;
```

Comment:
Copy the code from MOVDQA, which is better for `!MODREG` case.

Reply:
sorry, forgot  Ex as mem op.